### ISSUE TITLE
Cannot read property '$meta' of undefined — if src/main.js is renamed

### ISSUE THREAD

**Issue by Grawl** [universal-vue/uvue#45]
**Describe the bug**

I want to rename `src/main.js` to `index.js`. But if I change `paths.main` in `uvue.config.js` to `'./src/index.js'` I got this error:

```
 DONE  Compiled successfully in 195ms

 I  Server is running: http://localhost:8528
 I  Type "rs" to restart server
[1583206208615] ERROR (9890 on iMac.local): Cannot read property '$meta' of undefined
    TypeError: Cannot read property '$meta' of undefined
        at __webpack_exports__.default (webpack:///./node_modules/@uvue/core/lib/server.js?:30:18)
```

**To Reproduce**

1. Rename `src/main.js` to `index.js` or anything
2. Add `main: './src/index.js'` to `paths: {}` in `uvue.config.js`
3. Run `vue-cli-service ssr:serve`

**Additional context**

```
node: 12.15.0
vue-cli: 3.12.1
@uvue/core: 0.1.5
@uvue/server: 0.1.3
os: macOS Catalina 10.15.2 (19C57)
browser: Google Chrome 80.0.3987.122
```

**Comment by Grawl** [comment]
Actually, I want to move `server/` and `plugins/` out of `src/` and rename `src/` to `client/` to separate configurations, backend and frontend code.

**Comment by chymz** [comment]
Thanks for reporting this, indeed there is a little bug here, it will be fixed soon!

As a workarround, try to not provide the extension of your main file, (`.js` or `.ts`), so you can setup your `uvue.config.js` like this:

```js
export default {
  paths: {
    main: 'index', // Without file extension
  }
};
```

**Comment by Grawl** [comment]
Thank you! It's working for me now.

Also, I have to add `chain.resolve.alias.set('@', path.resolve(__dirname, 'client'))` into `chainWebpack()` in `vue.config.js`

**Comment by chymz** [comment]
Fixed in `0.1.6`

**Comment by robodude666** [comment]
Piggy backing off of google's top result to hopefully save someone else an hour. I've also ran into this error if I switch `module` to `commonjs` in tsconfig.json.
